packages/mobile/e2e/src/usecases/Settings.js

```diff
@@ -0,0 +1,31 @@
+import { dismissBanners } from '../utils/banners'
```

Comment:
Since this is wholly new it seems like a candidate for a separate PR.

Reply:
In the future, I'd like to have additional use cases in their own PR; however, as the tests are restructured I wanted to avoid submitting a PR which would break existing tests if merged.